priv/perf/apps/omg_load_testing/lib/utils/account.ex

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitation
+
+defmodule OMG.LoadTesting.Utils.Account do
+  @moduledoc """
+  Utility module for utxo
+  """
+
+  @doc """
+  Repeats f until f returns {:ok, ...}, :ok OR exception is raised (see :erlang.exit,:erlang.error) OR timeout
+  after `timeout` milliseconds specified
+
+  Simple throws and :badmatch are treated as signals to repeat
+  """
+  require Logger
+
+  alias ExPlasma.Encoding
+  alias OMG.LoadTesting.Utils.Crypto
+
+  @type private_key_t :: <<_::256>>
+  @type private_key_hex_t :: <<_::512>> | <<_::528>>
+  @type public_key_t :: <<_::512>>
+  @type addr_t :: <<_::160>>
+  @type t :: %__MODULE__{
+          priv: private_key_t(),
+          pub: public_key_t(),
+          addr: addr_t()
+        }
+
+  defstruct [:priv, :pub, :addr]
+
+  @spec new(private_key_t()) :: {:ok, t()} | {:error, atom()}
+  def new(private_key) when byte_size(private_key) == 32 do
+    with {:ok, der_public_key} <- compute_public_key(private_key),
```

Comment:
this with statement is defensive and hides if an error occurs. be explicit and bold in your statements. you want to crash as early as possible. remove the with and just assert exactly what you want it to be.